Dockerfile

```diff
@@ -4,7 +4,6 @@ COPY package.json package-lock.json ./
 RUN npm set progress=false && \
   npm config set depth 0 && \
   npm ci
-RUN npm audit --production --audit-level=moderate
```

Comment:
Hvorfor fjerne audit?

Reply:
Fordi den feiler :P

Reply:
Er dette pga. en audit som det ikke finnes fix til? I fdk-portal hadde vi samme problem for noen uker siden. Der finnes det en commit "chore: audit fix using npm-audit-resolver" som bytter til å bruke et bibliotek "npm-audit-resolver" som gjør det mulig å disable spesifikke bibliotek fra audit i en begrenset periode. Det er en løsning vi kan ta i bruk her også heller enn å disable.

Reply:
Ja, på en måte

Den feiler pga xmldom. De har fikset feilen, men får ikke til å publisere ny versjon til npm før original eiger av xmldom sier det er greit. De har en ny versjon i github-packages, som hadde gått greit om vi brukte xmldom direkte. Men vi har avhengigheten gjennom rdflib, så må i så fall vente til de oppdaterer til ny versjon hos seg.

Jeg skal sjekke ut npm-audit-resolver :+1:

Reply:
cfa4c18